### DISCUSSION TITLE
Quesition on ADF, blob storage for FinOps. MS Fabric

### DISCUSSION THREAD

**Discussion by ahernon** [microsoft/finops-toolkit#789] [General]
You must have the following permissions to create the [deployed resources](https://microsoft.github.io/finops-toolkit/hubs/template#%EF%B8%8F-resources).

Will this work with MS Fabrics ADF, Onelake (essentially Gen2 storage)?

Thank you

**Comment by flanakin** [comment]
Are you asking about connecting Power BI reports to OneLake or connecting Fabric to a hub instance or something else?

The hubs prereqs are for hubs only. If I remember correctly, Fabric only requires Storage Blob Data Reader to create a shortcut and read data from ADLS.

**Comment by jamelachahbar** [comment]
I have a test implementation running in fabric using shortcuts pointing to adls gen2. Yes, I can confirm this works. You can create your own data factory pipeline in Fabric but to my knowledge, there is no reference implementation of this yet for this project so you will probably need to recreate the steps yourself in Fabric Data factory. You can use the adls gen2 output from the datafactory pipeline (the ingestion folder containing all the parquet files)which can be consumed as is and create your dashboards in fabric. Unless you are looking to customize tables and do additional transformations in fabric you do not need to create a fabric datafactory pipeline.
